inflearn/onebit-react/section10/src/components/List.tsx

```
@@ -0,0 +1,23 @@
+}
+
+export const List = ({ todos }: Props) => {
+    return (
+        <div className="List">
+            <h4>Todo List</h4>
+            <input placeholder="검색어를 입력하세요." />
+            <div className='todos_wrapper'>  
+             {
+                todos.map((item) => (
+                    <TodoItem todo={item}/>
+                ))
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)